packages/react/src/components/ContentSwitcher/ContentSwitcher.js

```diff
@@ -67,14 +67,42 @@ export default class ContentSwitcher extends React.Component {
     );
   }
 
+  /**
+   * A "ring buffer" function that takes an array and depending on an ArrowRight
+   * or ArrowLeft key input loops from last index to first or first index to last.
+   */
+  getNextIndex = (key, index, arrayLength) => {
```

Comment:
Would you want to hoist this function outside the class given there seems no `this` reference in it?

Reply:
Talked with @joshblack about setting up an a11y tools file, so I just went ahead and did that 😄